dependency/inversion/principle/DBConnector.java

```diff
@@ -0,0 +1,14 @@
+package solid.dependency.inversion.principle;
+
+public class DBConnector implements DBOperations {
+
+	@Override
+	public void save(Object obj) {
+
+		//save the object toMySQL
+		
+	}
+	
+	
+
```

Comment:
Please remove the unused break line. Check the other as well

Reply:
I have adjusted